packages/driver/src/cypress/runner.ts

```diff
@@ -1597,7 +1597,9 @@ export default {
         )
       },
 
-      run (fn) {
+      run (fn, response) {
+        Cypress.__actions = response?.actions
```

Comment:
What is this used for?

Reply:
this is used to make the burn-in actions available for the mocha prototype extensions here https://github.com/cypress-io/cypress/blob/15f8fc35c8ba8519a3c7b090b60f0a6f005fe103/packages/driver/src/cypress/mocha.ts#L462